layouts/partials/posts_navigation.html

```diff
@@ -1,15 +1,23 @@
 {{ $skipPageNav := .Scratch.Get "skipPageNav" }}
 {{ if not (or .Params.skip_pagenav $skipPageNav) }}
-    {{ if or .NextInSection .PrevInSection }}
+    {{$prev := .PrevInSection}}
+    {{$next := .NextInSection}}
```

Comment:
```suggestion
    {{ $prev := .PrevInSection }}
    {{ $next := .NextInSection }}
```

Reply:
standard is spaces after/before the moustaches
(yes, that's what they are called!)